BEST-PRACTICES.md

```diff
@@ -17,15 +17,24 @@ We recommend the following coding practices for high quality contributions:
 - Prefer self-explanatory code as much as possible, but provide helpful comments for complex expressions and code blocks.
 - Add unit tests for any new or changed functionality, and update any existing tests that are impacted by your changes.
   - SUSHI uses [Jest](https://jestjs.io/) as a testing framework.
-  - To run the full test suite, run `npm test`. Ensure all tests are passing.
+  - Ensure all tests are passing. Ensure that code coverage of the new code is complete.
+  - To run the full test suite, run `npm test`.
+  - To review the test coverage report, run `npm run coverage` after running the full test suite.
```

Comment:
When these bullets are all at the same level, I feel like they are backwards. Like it's telling you to ensure the tests pass -- and _then_ telling you to run the tests (when you really need to run them before you can ensure they pass).

I think if you nest the last two bullets another level, then that fixes it (and makes it more consistent with some of the bulleting below). OTOH, if that's _too much_ nesting, I wonder if you should change the order of the bullets?

Reply:
Hm yeah I actually moved tha bullet between here and the end of the list a couple times. When it was at the bottom, it felt like it got lost at the bottom if you already knew how to run the tests/coverage and that those were just extra details that were nice to have if you needed them. But I also understand what you mean here. I think I'd rather not indent a third level, so I'm just going to move the bullet for checking the tests pass to the end.

Reply:
Changed the order in [7c4b6fe](https://github.com/FHIR/sushi/pull/1474/commits/7c4b6fee2205e44567d370c0c1b861c3ba6a4746)